jina/serve/runtimes/head/__init__.py

```diff
@@ -349,3 +352,19 @@ def _merge_metadata(self, metadata, uses_after_metadata, uses_before_metadata):
             for key, value in uses_after_metadata:
                 merged_metadata[key] = value
         return merged_metadata
+
+    async def _status(self, empty, context) -> jina_pb2.JinaInfoProto:
+        """
+        Process the the call requested and return the JinaInfo of the Runtime
+
+        :param empty: The service expects an empty protobuf message
+        :param context: grpc context
+        :returns: the response request
+        """
+        infoProto = jina_pb2.JinaInfoProto()
+        version, env_info = get_full_version()
+        for k, v in version.items():
+            infoProto.jina[k] = str(v)
+        for k, v in env_info.items():
+            infoProto.envs[k] = str(v)
+        return infoProto
```

Comment:
Can't we make that a Mixin than can be use in WorkerRuntime and HeadRuntime  ? Otherwise we duplicate code with WokerRuntime

Reply:
for such a small code, I do not want to add a mixin